website/docs/get-started/guides-overview.mdx

```diff
@@ -69,18 +69,18 @@ Learn more:
 Step-by-step examples for React, Java, Python, and other SDKs show you how to connect, configure, and evaluate feature flags in real applications.
 
 **Frontend SDKs**
-- [React](/feature-flag-tutorials/react)
-- [JavaScript](/feature-flag-tutorials/javascript)
-- [Next.js](/feature-flag-tutorials/nextjs)
-- [SvelteKit](/feature-flag-tutorials/sveltekit)
-- [Flutter](/feature-flag-tutorials/flutter/a-b-testing)
-- [iOS](/feature-flag-tutorials/ios)
+- [React](/guides/implement-feature-flags-in-react)
+- [JavaScript](/guides/implement-feature-flags-in-javascript)
+- [Next.js](/guides/implement-feature-flags-in-nextjs)
+- [SvelteKit](/guides/implement-feature-flags-in-sveltekit)
+- [Flutter](/guides/implement-ab-test-in-flutter)
+- [iOS](/guides/implement-feature-flags-in-ios)
 
 **Backend SDKs**
-- [Java](/feature-flag-tutorials/java)
-- [Python](/feature-flag-tutorials/python)
-- [Go](/feature-flag-tutorials/go)
-- [.NET](/feature-flag-tutorials/dotnet)
-- [Ruby](/feature-flag-tutorials/ruby)
-- [Rust](/feature-flag-tutorials/rust)
-- [Serverless](/feature-flag-tutorials/serverless/lambda)
\ No newline at end of file
+- [Java](/guides/implement-feature-flags-in-java)
+- [Python](/guides/implement-feature-flags-in-python)
+- [Go](/guides/implement-feature-flags-in-golang)
+- [.NET](/guides/implement-feature-flags-in-dotnet)
+- [Ruby](/guides/implement-feature-flags-in-ruby)
+- [Rust](/guides/implement-feature-flags-in-rust)
+- [Serverless](/guides/implement-feature-flags-in-aws-lambda)
```

Comment:
I'd just say "AWS Lambda" here since we don't cover any other serverless service/framework. And then we'd also need to update that page's title.

```suggestion
- [AWS Lambda](/guides/implement-feature-flags-in-aws-lambda)
```